cupy/math/window.py

```diff
@@ -87,3 +92,51 @@ def hanning(M):
         return basic.ones(1, float)
     n = ranges.arange(0, M)
     return 0.5 - 0.5 * trigonometric.cos(2.0 * numpy.pi * n / (M - 1))
+
+
+@util.memoize()
+def _kaiser_kernel():
+    input_arguments = "float32 beta, float32 alpha"
```

Comment:
Since the kernel does not have any templated parameters,
it is just better to have it in a global variable

```python
_kaiser_kernel = ElementwiseKernel(
    'float32 beta, float32 alpha'
    'T arr'
    ...
)
```
(Thanks to @asi1024 )

Reply:
Done